### DISCUSSION TITLE
[Support]: Upgrading Frigate Proxmox container from 0.13.2 to 0.14

### DISCUSSION THREAD

**Discussion by ChienDeLaCasse** [blakeblackshear/frigate#12203] [Ask A Question] [answered]
### What is your question:

Greetings, 
sorry for all those basic questions. I am still quite the beginner in Linux based environments.

My Frigate is currently 0.13.2, as a container under Proxmox (installed thanks to the famous and much useful "VE helper scripts". 
I would like to use the 0.14. 

I have saved my config file ans also my frigate.db.

Is there any way to "upgrade my container" ? Or do I have to wipe it and make a fresh install (then import config and frigate.db) ?

Many thanks in advance for your support !

**Comment by NickM-27** [comment, marked as answer]
that would be a question for the creator of the VE helper scripts. That is maintained by someone else and we have no knowledge of how it works or how you would go about updating

**Reply by ChienDeLaCasse** [reply]
Thank you !
